contracts/contract-webauthn-secp256r1/src/lib.rs

```diff
@@ -219,20 +188,20 @@ impl CustomAccountInterface for Contract {
         signature: Signature,
         auth_contexts: Vec<Context>,
     ) -> Result<(), Error> {
-        // Only the sudo signer can `add_sig`, `rm_sig`, `resudo` and `upgrade`
+        // Only the super signer can `add_sig`, `rm_sig`, `re_super` and `upgrade`
         for context in auth_contexts.iter() {
             match context {
                 Context::Contract(c) => {
                     if c.contract == env.current_contract_address()
                         && (c.fn_name == Symbol::new(&env, "add_sig")
                             || c.fn_name == Symbol::new(&env, "rm_sig")
-                            || c.fn_name == Symbol::new(&env, "resudo")
+                            || c.fn_name == Symbol::new(&env, "re_super")
                             || c.fn_name == Symbol::new(&env, "upgrade"))
                     {
                         if env
                             .storage()
                             .instance()
-                            .get::<Symbol, Bytes>(&SUDO_SIGNER)
+                            .get::<Symbol, Bytes>(&SUPER)
                             .ok_or(Error::NotFound)?
                             != signature.id
                         {
```

Comment:
`_ => return Err(Error::InvalidContext),` - that's disallows deploying new contracts by this account, which seems like a weird/arbitrary restriction to me.

Reply:
Are there any concerns such that only the super signer should be allowed to deploy new contracts? Probably not. So this should just be `_ => {}` to enable it correct? Essentially pass over that instance.

Reply:
Yes, just _ => {} is good, my point is that we shouldn't really return an error for authorizing the host functions.